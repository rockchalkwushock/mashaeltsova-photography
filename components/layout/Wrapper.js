const Wrapper = ({ children }) => (
  <div className="wrapper">
    {children}
    <style jsx>{`
      .wrapper {
        display: -webkit-box;
        display: -ms-flexbox;
        display: flex;
        -webkit-box-orient: vertical;
        -webkit-box-direction: normal;
        -ms-flex-direction: column;
        flex-direction: column;
        min-height: 100vh;
        text-align: center;
      }
    `}</style>
  </div>
)

export default Wrapper
