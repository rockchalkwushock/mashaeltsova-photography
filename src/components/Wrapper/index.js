import styled from 'styled-components'

const Wrapper = styled.div`
  display: flex;
  flex-direction: column;
  min-height: 100vh;
`

Wrapper.displayName = 'Wrapper'

export default Wrapper
