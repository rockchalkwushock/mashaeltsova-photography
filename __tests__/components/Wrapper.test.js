import React from 'react'
import { render } from 'enzyme'

import Wrapper from '../../components/commons/Wrapper'

describe('Component: Wrapper', () => {
  test('Renders without exploding', () => {
    const tree = render(<Wrapper />)
    expect(tree).toMatchSnapshot()
  })
})
