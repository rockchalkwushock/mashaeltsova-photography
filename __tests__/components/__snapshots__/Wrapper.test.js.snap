// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Wrapper Renders without exploding 1`] = `
<div
  class="jsx-3955059306 wrapper"
/>
`;
